### PR TITLE
BACKLOG-5664 Repurpose PentahoAwareCharacterEncodingFilter to presenе static html page

### PR DESCRIPTION
**Pull request by AliaksandrDrebenchuk**
SystemInitStatusFilter implemented
PentahoAwareCharacterEncodingFilter renamed to PentahoAwareCharacterEncodingFilterImpl
current PentahoAwareCharacterEncodingFilter work as wrap for PentahoAwareCharacterEncodingFilterImpl and SystemInitStatusFilter
ServerStatusChangeListenerList synchronized

### CODE REVIEW

**Comment by AliaksandrDrebenchuk** [comment]
Current problems:
 - Resources(images) not available for the first opening of the page.
 - 'Busy' page don't contains algoritm for autorefresh.

**Comment by AliaksandrDrebenchuk** [comment]
@rmansoor  @pedrofvteixeira please review

File: assembly/package-res/biserver/tomcat/webapps/pentaho/html/InitState.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+					<tr>
+						<td width="18px" />
+						<td style="border-bottom:2px solid #EFEFEF" />
+						<td width="18px" />
+					</tr>
+					<tr/>
+					<tr>
+						<td width="18px" />
+						<td class="text-body">
+							We're sorry, the Pentaho User Console is not available now. Please try logging in agin later.
```
Comment (pedrofvteixeira) [inline assembly/package-res/biserver/tomcat/webapps/pentaho/html/InitState.html]: > "(...) Please try logging in agin later."

Typo: "again"

**Comment by pedrofvteixeira** [comment]
Tested locally on my machine, all seems ok. The "Busy screen" static page appeared as expected. 
Quick-fix the typo, and from my part it's good to go.

**Comment by rmansoor** [comment]
@AliaksandrDrebenchuk processing the pull request now. Pedro T will update with the typo fix

**Comment by rmansoor** [comment]
@AliaksandrDrebenchuk auto refresh was never part of this story. So the only remaining issue is the styling one

**Pull request merged**